pkg/operator/ceph/object/rgw.go

```diff
@@ -82,8 +82,12 @@ func (c *clusterConfig) createOrUpdateStore(realmName, zoneGroupName, zoneName s
 		return nil
 	}
 
-	if err = enableRGWDashboard(objContext); err != nil {
-		logger.Warningf("failed to enable dashboard for rgw. %v", err)
+	if c.clusterSpec.Dashboard.Enabled {
```

Comment:
I think only check the value of the cr is not enough, we need to check whether the dashboard  is running normal. if the dashboard enable failed, no need to enable rgw dashboard.

Reply:
> The changes look good, could you just add more details to the commit message? Perhaps like this:
> 
> ```
> rgw: allow the rgw dashboard to be disabled
> 
> The rgw dashboard can be disabled by the setting in the 
> CephObjectStore `gateway.dashboardEnabled` if set to false.
> ```

done